src/rocksdb.rs

```diff
@@ -1664,7 +1664,7 @@ impl Drop for DB {
     fn drop(&mut self) {
         // SyncWAL before call close.
         if !self.readonly {
-            self.sync_wal().unwrap();
+            self.sync_wal().unwrap_or_else(|_| {});
```

Comment:
Why?

Reply:
not all wal support sync, such as wal created by mem env

Reply:
Then you should add some comments here.